frontend/www/js/omegaup/arena/contest_practice.ts

```diff
@@ -185,19 +191,24 @@ OmegaUp.on('ready', () => {
               message: clarification.message,
             })
               .then(() => {
-                target.clearForm();
-                refreshClarifications({ clarification, contestAlias, target });
+                clearForm;
```

Comment:
```suggestion
                clearForm();
```

aquí hay que mandarlo llamar o no estaría haciendo nada.

Reply:
Listo